.github/workflows/build_deploy.yml

```diff
@@ -136,3 +136,59 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.PROD_AWS_SECRET_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}
       # </beta.app.ssv.network>
+
+      # <faucet.ssv.network>
+      - name: Run prod faucet build
+        if: github.ref == 'refs/heads/main'
+        run: >
+          SSV_NETWORKS="${{ env.STAGE_SSV_NETWORKS }}"
+          GOOGLE_TAG_SECRET="$GOOGLE_TAG_SECRET"
+          EXPLORER_URL="$EXPLORER_URL" BLOCKNATIVE_KEY="$BLOCKNATIVE_KEY" yarn build
+        env:
+          REACT_APP_FAUCET_PAGE: "true"
+          SSV_NETWORKS: ${{ env.STAGE_SSV_NETWORKS }}
+          BLOCKNATIVE_KEY: ${{ secrets.BLOCKNATIVE_KEY }}
+          EXPLORER_URL: "https://explorer.ssv.network/"
+          LINK_SSV_DEV_DOCS: ${{ secrets.LINK_SSV_DEV_DOCS }}
+          REACT_APP_CAPTCHA_KEY: ${{ secrets.CAPTCHA_KEY_STAGE }}
+      - name: Deploy prod faucet
+        if: github.ref == 'refs/heads/main'
+        uses: jakejarvis/s3-sync-action@v0.5.0
+        with:
+          args: --acl public-read --follow-symlinks --delete
+        env:
+          SOURCE_DIR: 'build/'
+          AWS_REGION: 'us-west-2'
+          AWS_S3_BUCKET: ${{ secrets.PROD_FAUCET_AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.PROD_AWS_SECRET_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.PROD_AWS_SECRET_ACCESS_KEY }}
+      # </faucet.ssv.network>
+
+      # <claim.ssv.network>
+      - name: Run prod build claim
+        if: github.ref == 'refs/heads/main'
+        run: >
+          SSV_NETWORKS="${{ env.STAGE_SSV_NETWORKS }}"
+          GOOGLE_TAG_SECRET="$GOOGLE_TAG_SECRET"
+          EXPLORER_URL="$EXPLORER_URL" BLOCKNATIVE_KEY="$BLOCKNATIVE_KEY" PROD_CLAIM_PAGE="$PROD_CLAIM_PAGE" yarn build
+        env:
+          PROD_CLAIM_PAGE: "true"
+          BLOX_INFRA_API: 'https://api.bloxstaking.com'
+          BLOCKNATIVE_KEY: ${{ secrets.BLOCKNATIVE_KEY }}
+          EXPLORER_URL: "https://explorer.shifu.ssv.network"
+          LINK_SSV_DEV_DOCS: ${{ secrets.LINK_SSV_DEV_DOCS }}
+          GOOGLE_TAG_SECRET: ${{ secrets.PROD_GOOGLE_TAG_SECRET }}
```

Comment:
You have `ssv_networks.explorerUrl`
You have `ssv_networks.googleTagSecret `

Reply:
done